package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "blog-test",
+  "name": "remix-movies",
```

Comment:
Hope you don't mind the two name changes. This name matches the recommended Cloudflare Pages name in the Readme.

As for the D1 database, I figured `pokemon` might be a little confusing 😜